app/cdap/text/text-en.yaml

```diff
@@ -5,6 +5,9 @@ commons:
   apply: Apply
   as: as
   back: Back
+  learnMore: Learn More
+  gotIt: Ok, Got it.
+  cookieBanner: CDF uses cookies from Google to deliver and enhance the quality of its services and to analyze traffic.
```

Comment:
Is this only valid for CDF? Or could it appear for CDAP?

Reply:
Yeah it could appear in cdap but it also gives the option for someone else to overwrite it using a config - I weighed the options and this seemed to be the most prudent since this can easily be translated from here?

Reply:
The only thing I could propose is to replace "CDF" with "This application" to make it apply to everything. But I don't think it's worth holding this up further.